.prettierrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "bracketSpacing": false
+}
```

Comment:
Maybe we should add prettier to the `package.json` now. I don't know whether you forgot or just have it installed globally, but I think it would be beneficial to everyone working on this repo.

Reply:
Would adding a CI check or commit hook also be helpful?

Reply:
I'm skipping further Prettier tweaks in this PR.

See #263